cmake/scripts/FindVersion.cmake

```diff
@@ -124,6 +124,19 @@ if(GENERATE_OTTDREV)
     message(STATUS "Generating .ottdrev")
     file(WRITE ${CMAKE_SOURCE_DIR}/.ottdrev "${REV_VERSION}\t${REV_ISODATE}\t${REV_MODIFIED}\t${REV_HASH}\t${REV_ISTAG}\t${REV_ISSTABLETAG}\t${REV_YEAR}\n")
 else()
+    string(REGEX MATCH "^[0-9.]+" REV_SPLIT "${REV_VERSION}")
```

Comment:
Maybe use `REV_ISSTABLETAG` here to know if the version is expected to be a stable tag? No clue if that is easy to do :)

Reply:
Could work, beta and RC would then default to hard coded version, which should not be an issue.